features/prompt-registry.mdx

```diff
@@ -97,6 +97,27 @@ Alternatively, use the REST API endpoint `/rest/track-prompt` ([read more](/refe
 
 Learn more about tracking templates [here](/features/prompt-history/tracking-templates)
 
+
+## Tracking Groups
```

Comment:
This doesn't seem true to me? How is Prompt Registry integrated with groups?

Reply:
Please remove unless I'm wrong

Reply:
It might not be strictly related, so I'm removing it.